common/common.env

```diff
@@ -20,13 +20,19 @@ S3_SOURCE_DATA_PATH=/source-data
 S3_DATASET_PATH=/auto-generated/datasets
 ASGS2016_DATASET_FILE=asgs2016.trig.gz
 ASGS2011_DATASET_FILE=asgs2011.trig.gz
-GEOFABRIC_DATASET_FILE=geofabric.trig.gz
+GEOFABRIC_DATASET_FILE=geofabric2.trig.gz
 GNAF_DATASET_FILE=gnaf_current.trig.gz
-GNAF1605_DATASET_FILE=gnaf_1605.trig.gz
+GNAF1605_ADDRESS_DATASET_FILE=gnaf_201605_address_instances.nt.gz
+GNAF1605_LOCALITY_DATASET_FILE=gnaf_201605_locality_instances.nt.gz
+GNAF1605_STREET_LOCALITY_DATASET_FILE=gnaf_201605_streetLocality_instances.nt.gz
+GNAF1605_ADDRESS_SITES_DATASET_FILE=gnaf_201605_addressSites_instances.nt.gz
```

Comment:
Not particularly excited this file is being split up. It will make the GNAF harverster a bit more complicated to write, and is presumably only done like this because of a short-coming of the manual harvest process of the gnaf (ie, not being able to zip all files together).